src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py

```diff
@@ -247,12 +247,13 @@ class CtsDemandBuildings(Dataset):
     def __init__(self, dependencies):
         super().__init__(
             name="CtsDemandBuildings",
-            version="0.0.0",
+            version="0.0.1",
             dependencies=dependencies,
             tasks=(
                 cts_buildings,
                 {cts_electricity, cts_heat},
                 {get_cts_electricity_peak_load, get_cts_heat_peak_load},
+                assign_voltage_level_to_buildings,
```

Comment:
I wasn't able to find the task in the current run but it's in the server's codebase :thinking: 
How come?

Reply:
Git blames myself. Apparently this task and the mapping dissolved by merging them :(